heron/config/src/yaml/conf/yarn/uploader.yaml

```diff
@@ -1,5 +1,5 @@
 # uploader class for transferring the topology jar/tar files to storage
-heron.class.uploader:                            com.twitter.heron.uploader.NullUploader
+heron.class.uploader:                            com.twitter.heron.uploader.localfs.LocalFileSystemUploader
```

Comment:
I've changed back to `NullUploader` for now, but the proper fix is really to change the yarn impl (or the shared classes it relies on) to not invoke an uploader at all. This is a cleaner approach than calling a class written to do nothing.